lib/player.js

```diff
@@ -2517,6 +2521,9 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       if (this.config_.streaming.startAtSegmentBoundary) {
         const timeline = this.manifest_.presentationTimeline;
         let initialTime = this.startTime_ || this.video_.currentTime;
+        if (!this.startTime_ && this.manifest_.startTime) {
```

Comment:
The `startTime` parameter on `load()` could be 0. You should instead check `if (this.startTime_ == null ...`

Reply:
Done!